README.md

```diff
@@ -40,6 +40,11 @@ If you don't want to run the `bundle` command after bumping, add the `--no-bundl
     
     bump patch --no-bundle
 
+### --no-berks
+If you don't want to run the `berks` command after bumping, add the `--no-berks` option.
```

Comment:
bundler is automatic too ? should berks not be (if there is a Berksfile) ?

On Mon, Jun 20, 2016 at 9:39 AM, Gregory <notifications@github.com> wrote:

> In README.md
> <https://github.com/gregorym/bump/pull/55#discussion_r67722550>:
>
> > @@ -40,6 +40,11 @@ If you don't want to run the `bundle` command after bumping, add the `--no-bundl
> >
> >      bump patch --no-bundle
> >
> > +### --no-berks
> > +If you don't want to run the `berks` command after bumping, add the `--no-berks` option.
>
> It should be the other way around. Pass a flag if you want to run berks
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/gregorym/bump/pull/55/files/eecb5ace00e81577e21313cb5a0a21a1599cbed4#r67722550>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ6Ei8uXGGBXYC3VSXyY-IAtqoYaBks5qNsJYgaJpZM4I5v51>
> .
>